source/jsonnet/northern-ireland/individual/blocks/identity-and-health/country_of_birth.jsonnet

```diff
@@ -70,20 +70,53 @@ local question(title) = {
   routing_rules: [
     {
       goto: {
-        block: 'arrive-in-country',
+        block: 'passports',
         when: [
           {
             id: 'country-of-birth-answer',
-            condition: 'not equals',
-            value: 'Northern Ireland',
+            condition: 'not set',
+          },
+          rules.under1,
+        ],
+      },
+    },
+    {
+      goto: {
+        block: 'past-usual-household-address',
+        when: [
+          {
+            id: 'country-of-birth-answer',
+            condition: 'not set',
           },
         ],
       },
     },
     {
       goto: {
         block: 'passports',
-        when: [rules.under1],
+        when: [
+          {
+            id: 'country-of-birth-answer',
+            condition: 'set',
+          },
+          rules.under1,
+        ],
+      },
+    },
```

Comment:
These says go to passport if country of birth is unanswered and under 1 or country of birth is set and under 1. So it should just be, go to passport when under 1.